src/ui/src/builder/journal/BuilderJournalSearchbar.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+	<div class="BuilderJournalSearchbar">
+		<WdsTextInput
+			v-model="searchText"
+			class="searchInput"
+			left-icon="search"
+			@input="onSearchChange"
+		/>
+		<div class="BuilderJournalSearchbar__dropdown">
+			<WdsDropdownMenu
+				:selected="statuses"
+				:options="statusOptions"
+				:enable-multi-selection="true"
+				@select="onStatusesChange"
+			></WdsDropdownMenu>
+		</div>
+		<div class="BuilderJournalSearchbar__dropdown">
+			<WdsDropdownMenu
+				:selected="triggers"
+				:options="triggerOptions"
+				:enable-multi-selection="true"
+				@select="onTriggersChange"
+			></WdsDropdownMenu>
+		</div>
+	</div>
+</template>
+
+<script setup lang="ts">
+import WdsDropdownMenu from "@/wds/WdsDropdownMenu.vue";
+import WdsTextInput from "@/wds/WdsTextInput.vue";
+import { Ref, ref } from "vue";
+
+const statusOptions = [
+	{ value: "success", label: "Success" },
+	{ value: "error", label: "Error" },
+	{ value: "stopped", label: "Stopped" },
+];
+const triggerOptions = [
+	{ value: "On demand", label: "On demand" },
+	{ value: "UI", label: "UI" },
+	{ value: "API", label: "API" },
+	{ value: "Cron", label: "Scheduled" },
+];
+
+const searchText = ref("");
+const statuses: Ref<string[]> = ref([]);
+const triggers: Ref<string[]> = ref([]);
+
+const emit = defineEmits<{
+	(e: "update:search", value: string): void;
+	(e: "update:statuses", value: string[]): void;
+	(e: "update:triggers", value: string[]): void;
+}>();
+
+function onSearchChange() {
+	emit("update:search", searchText.value);
+}
+
+function onStatusesChange(val: string[]) {
+	statuses.value = val;
+	emit("update:statuses", val);
+}
+
+function onTriggersChange(val: string[]) {
+	triggers.value = val;
+	emit("update:triggers", val);
+}
```

Comment:
I think you want to use `v-model` here. It creates the emits logic for you
```suggestion
const searchText = defineModel("search", {type: String});
const statuses = defineModel("statuses", {type: Array as PropType<string[]>});
const triggers = defineModel("triggers", {type: Array as PropType<string[]>});

function onStatusesChange(val: string[]) {
	statuses.value = val;
}

function onTriggersChange(val: string[]) {
	triggers.value = val;
}
```